src/MuTalk-Tests/MTRandomClassMutantSelectionStrategyTest.class.st

```diff
@@ -0,0 +1,12 @@
+Class {
+	#name : 'MTRandomClassMutantSelectionStrategyTest',
+	#superclass : 'MTRandomMutantSelectionStrategyTest',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'accessing' }
+MTRandomClassMutantSelectionStrategyTest >> classToTest [
+
+	^ MTRandomClassMutantSelectionStrategy
+]
```

Comment:
This class do not define new tests?

Reply:
I agree, but I struggle a bit to see what kind of tests I could do, given the random nature of the selection.
I guess I could create an auxiliar class for each one, like a class with let's say 4 methods, which produce enough mutants to be able to have around 25% of mutants from each method with the random method mutant selection.

Reply:
Or else I could create a set of auxiliar classes for all random selections, and given a seed (or maybe even not), I check that I get a different mutant selection with each strategies

Reply:
Additionally, we can see if we can parametrise the random selector with a seed, and use a same fixed seed always in the test, to guarantee we have the same results.

Reply:
I would combine both your ideas: (1) a class with a list of selectors that we control easily, and (2) the seed. :)

Reply:
Ok thanks !